documentation/api/reference/made_with_ov.md

```diff
@@ -0,0 +1,76 @@
+# Made with Openverse
+
+The Openverse API provides access to the world's largest library of
+openly-licensed creative works, containing over 700 million images and audio
+tracks. All Openverse content is under a
+[Creative Commons license](https://creativecommons.org/licenses/) or is in the
+public domain.
+
+As a developer, you can use the Openverse API to create incredible apps. Here
+are some apps powered by the Openverse API.
+
+```{note}
+If you've made something using the Openverse API that you would like to share,
+[let us know](https://github.com/WordPress/openverse#keep-in-touch) and we'll
+feature it here!
+```
+
+## Openverse.org
+
+The [Openverse search engine](https://openverse.org) is the primary user
+interface to the Openverse library. It is also the most actively developed and
+the first to feature new media types.
+
+**Available media:** images, audio
+
+![Openverse.org homepage](/_static/openverse_homepage.webp)
+
+## Raycast extension
+
+Openverse's [Raycast extension](https://www.raycast.com/openverse/openverse)
+makes it extremely convenient to find, use and properly attribute images, all
+right from your desktop.
+
+**Available media:** images
+
+![Raycast extension](/_static/openverse_raycast.webp)
+
+## Slack extension
+
+Openverse's
+[Slack extension](https://openverse-slack.sarayourfriend.pictures/slack/)
+provides a fun and easy way to search and insert Openverse images into Slack
+conversations, with attribution of course.
+
+**Available media:** images
+
+<img src="/_static/openverse_slack.webp" width="335" height="541" />
```

Comment:
Why do we have to specify the dimensions here, if we don't do that for all the other images?

Reply:
This one is a very narrow and tall image. If the dimensions are not specified, it tries to occupy the full width and blows up in height.

Reply:
TIL! Fixed in ca27a4a.